pkg/processor/trigger/kafka/factory.go

```diff
@@ -45,7 +45,7 @@ func (f *factory) Create(parentLogger logger.Logger,
 
 	// create worker allocator
 	workerAllocator, err := worker.WorkerFactorySingleton.CreateFixedPoolWorkerAllocator(kafkaLogger,
-		32, // TODO: Allocate dynamically.
+		configuration.MaxWorkers, // TODO: Allocate dynamically.
```

Comment:
no need for the TODO comment anymore

Reply:
@sahare92 not true, the comment is still relevant as maybe in the future we would want to add a dynamic pool worker allocator independently to user's input